### PR TITLE
Update strings.xml -ru

### PR DESCRIPTION
**Pull request by Anready**

### CODE REVIEW

File: app/src/main/res/values-ru/strings.xml [hunk 1]
```diff
@@ -215,7 +215,7 @@
     <string name="error_no_matching_directory_found">Ошибка! Соответствующий каталог не найден!</string>
     <string name="cmd_clear_help">Очищает консоль</string>
     <string name="cmd_takes_no_params">\'%1$s\' команда не принимает никаких параметров.</string>
-    <string name="cmd_cmdrequest_help">Если вы хотите добавить новые команды в Yantra Launcher, выполните эту команду, чтобы открыть шаблон электронного письма для запроса команды. Имейте ввиду, что выполнение запроса на добавление команды не гарантируется, но я постараюсь сделать все возможное, чтобы включить ее в будущие версии Yantra Launcher, если она будет подходящей.</string>
+    <string name="cmd_cmdrequest_help">Если вы хотите добавить новые команды в Yantra Launcher, выполните эту команду, чтобы открыть Github Issues для запроса команды. Имейте ввиду, что выполнение запроса на добавление команды не гарантируется, но я постараюсь сделать все возможное, чтобы включить ее в будущие версии Yantra Launcher, если она будет подходящей.</string> <!-- ага, например остановка внешних процессов невозможно впринципе -->
```
Comment (coderGtm) [inline app/src/main/res/values-ru/strings.xml:218]: why this comment??

File: app/src/main/res/values-ru/strings.xml [hunk 2]
```diff
@@ -542,7 +542,7 @@
     <string name="playing_previous_track">Воспроизведение предидущего трека...</string>
     <string name="invalid_parameter_see_help_to_see_usage_info">Неверный параметр! Смотрите \'help %1$s\' чтобы посмотреть информацию об использовании.</string>
     <string name="suggestion_bar_color">Цвет панели предложений</string>
-    <string name="modern_prompt_design">Modern Prompt Design</string>
-    <string name="import_local_font">Import Local Font</string>
-    <string name="select_a_font_file">Select a Font File</string>
+    <string name="modern_prompt_design">Новый дизайн promt</string>
```
Comment (coderGtm) [inline app/src/main/res/values-ru/strings.xml:545]: Modern, not new! Plus the prompt spelling

File: app/src/main/res/values-ru/strings.xml (hunk 1 above)
Comment (Anready) [reply app/src/main/res/values-ru/strings.xml:218]: because its needed at least for joke

File: app/src/main/res/values-ru/strings.xml (hunk 2 above)
Comment (Anready) [reply app/src/main/res/values-ru/strings.xml:545]: > Modern, not new! Plus the prompt spelling

okay, if "модернизированный вид промпта" its okay for length...

Comment (Anready) [reply app/src/main/res/values-ru/strings.xml:545]: > Modern, not new! Plus the prompt spelling

okay, if "модернизированный вид промпта" its okay for length...

**Comment by Anready** [comment]
All is okay now?

**Review by coderGtm — APPROVED** [review]

**Pull request merged**